src/networks.json

```diff
@@ -769,5 +769,45 @@
     "rpc": ["https://rpc.api.moonbeam.network"],
     "explorer": "https://blockscout.moonbeam.network",
     "start": 171135
-    }
+  },
+  "81": {
+    "key": "81",
+    "name": "Shibuya Network",
+    "shortName": "Shibuya",
+    "chainId": 81,
+    "network": "testnet",
+    "testnet": true,
+    "multicall": "0x3E90A35839ff0Aa32992d33d861f24dC95BBf74d",
+    "rpc": ["https://rpc.shibuya.astar.network:8545"],
+    "explorer": "https://blockscout.com/shibuya",
+    "start": 856303
+  },
+  "336": {
+    "key": "336",
+    "name": "Shiden Network",
+    "shortName": "Shiden",
+    "chainId": 336,
+    "network": "mainnet",
+    "multicall": "0x3E90A35839ff0Aa32992d33d861f24dC95BBf74d",
+    "rpc": [
+      "https://rpc.shiden.astar.network:8545",
+      "https://shiden.api.onfinality.io/public"
```

Comment:
Hi @bLd75 

There are few errors with these nodes, maybe they are not archive?

```
Errors for: https://rpc.shiden.astar.network:8545
fullArchiveNode Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32603,\"message\":\"fetch runtime chain id failed: Application(Execution(Other(\\\"Exported method EthereumRuntimeRPCApi_account_basic is not found\\\")))\"},\"id\":44}\n", error={"code":-32603}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x3e90a35839ff0aa32992d33d861f24dc95bbf74d\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://rpc.shiden.astar.network:8545", code=SERVER_ERROR, version=web/5.4.0)

Errors for: https://shiden.api.onfinality.io/public
fullArchiveNode Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32603,\"message\":\"fetch runtime chain id failed: Application(Execution(Other(\\\"Exported method EthereumRuntimeRPCApi_account_basic is not found\\\")))\"},\"id\":44}\n", error={"code":-32603}, requestBody="{\"method\":\"eth_getBalance\",\"params\":[\"0x3e90a35839ff0aa32992d33d861f24dc95bbf74d\",\"0x1\"],\"id\":44,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="https://shiden.api.onfinality.io/public", code=SERVER_ERROR, version=web/5.4.0)
```

Reply:
Hi @ChaituVR thanks for your review, we just noticed we have a problem on the getBalance method when you query from block 1 because we activated EVM (called Frontier pallet in Polkadot/Substrate framework) later on.
This will be fixed with Shiden as soon as we push this PR and upgrade the runtime (next week much probably), I will keep you up-to-date.
https://github.com/AstarNetwork/Astar/pull/564